docs/samples/logger/local/README.md

```diff
@@ -9,14 +9,16 @@ docker run -it -p 8000:80 --rm -t mendhak/http-https-echo
 Start an SKLearn Iris model on port 8080. You will need to have pip installed the sklearnserver. See `/python/sklearnserver`.
 
 ```
-python -m sklearnserver --model_dir gs://kfserving-samples/models/sklearn/iris --model_name sklearn-iris --http_port 8080
+gsutil cp -r gs://kfserving-samples/models/sklearn/iris .
```

Comment:
Is this a good idea? Why not read directly from GCS?

Reply:
I just did this as a convenience as I got a gcs auth issue with the python library on my local machine. I will try it again and see exactly what I hit.

Reply:
There is no auth on those files, so if that's the case, the python sdk is busted.

Reply:
Hitting https://github.com/googleapis/google-cloud-python/issues/3100 / https://blog.timekit.io/google-oauth-invalid-grant-nightmare-and-how-to-fix-it-9f4efaf1da35 Could be my clock

Reply:
oof.